packages/api-client/index.ts

```diff
@@ -68,6 +68,9 @@ class APIClient {
       parseQuestionDates(question);
       return question;
     },
+    notify: async (questionId: number): Promise<void> => {
```

Comment:
Can this just be `: void` instead? Not sure if that messes something up

Reply:
it gotta be a promise